runtime/web_worker.rs

```diff
@@ -137,7 +139,8 @@ pub struct WebWorkerOptions {
   pub args: Vec<String>,
   pub debug_flag: bool,
   pub unstable: bool,
-  pub ca_filepath: Option<String>,
+  pub ca_file: Option<String>,
```

Comment:
Why is this needed in the `WebWorkerOptions`? Can't we read the file during initialization?